minio/signer.py

```diff
@@ -171,7 +176,9 @@ def get_signed_headers(headers):
 def sign_v4(method, url, region, headers=None,
             credentials=None,
             content_sha256=None,
-            request_datetime=None):
+            request_datetime=None,
+            service="s3"
```

Comment:
These changes regarding signing look good to me

Reply:
Great, to simplify this pull request I've opened a PR with just the signing changes - https://github.com/minio/minio-py/pull/893